pages/dkp/kommander/1.4/workspaces/create-workspaces/index.md

```diff
@@ -22,6 +22,17 @@ From the workspace selection dropdown at the top of the screen, select **Create
 ![Workspace Form Advanced Annotations](/dkp/kommander/1.4/img/workspace-annotations.png)
 <br />_Create Workspace Advanced Options_
 
-When creating or editing a workspace, you can use the Advanced Options to add, edit, or delete annotations and labels to your workspace. Both the annotations and labels are applied to the workspace namespace.
+When creating or editing a workspace, you can use the Advanced Options to add, edit, or delete annotations and labels to your workspace. Both the annotations and labels are applied to the workspace's namespace.
 
 Labels that are added to a workspace, are also applied to all of the clusters in the workspace.
+
+You can also create a **custom namespace** for the Workspace you are creating.
+By default, a unique ID Namespace will be auto-generated based on the first 57 characters of your workspace name plus 5 unique alphanumeric characters.
+![Workspace default naming](/dkp/kommander/1.4/img/workspace-default-name.png)
+<br />_Workspace namespace fields_
+
+To override this default behavior, add what you specifically want to be for Workspace's namespace.
+![Workspace default naming](/dkp/kommander/1.4/img/workspace-custom-name.png)
+<br />_Adding a custom namespace for your Workspace_
+
+<p class="message--warning"><strong>NOTE: </strong>By setting a custom ID for the Namespace, Kommander cannot guarantee safety and uniqueness of the namespace across all clusters, and that it may override any namespaces of the same name on imported clusters.</p>
```

Comment:
```suggestion
<p class="message--warning"><strong>NOTE: </strong>Setting a custom ID for the Namespace in Kommander  does not guarantee safety and uniqueness of the namespace across all clusters. This may override any namespaces, with the same name, on imported clusters.</p>
```

Reply:
```suggestion
<p class="message--warning"><strong>NOTE: </strong>Setting a custom ID for the namespace in Kommander does not guarantee safety and uniqueness of the namespace across all clusters. This may override any namespaces, with the same name, on attached clusters.</p>
```
(minor edits on russ's suggestion)

Reply:
Thanks both @gracedo @russurquhart1